doc/classes/InputEventMouseMotion.xml

```diff
@@ -22,12 +23,13 @@
 		</member>
 		<member name="relative" type="Vector2" setter="set_relative" getter="get_relative" default="Vector2(0, 0)">
 			The mouse position relative to the previous position (position at the last frame).
-			[b]Note:[/b] Since [InputEventMouseMotion] is only emitted when the mouse moves, the last event won't have a relative position of [code]Vector2(0, 0)[/code] when the user stops moving the mouse.
+			[b]Note:[/b] Since [InputEventMouseMotion] may only be emitted when the mouse moves, checking for a relative position of [code]Vector2(0, 0)[/code] can't reliably detect when the mouse has stopped moving. A separate [SceneTreeTimer] will likely need to be used to detect that the mouse hasn't moved for some period of time.
 			[b]Note:[/b] [member relative] is automatically scaled according to the content scale factor, which is defined by the project's stretch mode settings. This means mouse sensitivity will appear different depending on resolution when using [member relative] in a script that handles mouse aiming with the [constant Input.MOUSE_MODE_CAPTURED] mouse mode. To avoid this, use [member screen_relative] instead.
 		</member>
 		<member name="screen_relative" type="Vector2" setter="set_screen_relative" getter="get_screen_relative" default="Vector2(0, 0)">
 			The unscaled mouse position relative to the previous position in the coordinate system of the screen (position at the last frame).
-			[b]Note:[/b] Since [InputEventMouseMotion] is only emitted when the mouse moves, the last event won't have a relative position of [code]Vector2(0, 0)[/code] when the user stops moving the mouse. This coordinate is [i]not[/i] scaled according to the content scale factor or calls to [method InputEvent.xformed_by]. This should be preferred over [member relative] for mouse aiming when using the [constant Input.MOUSE_MODE_CAPTURED] mouse mode, regardless of the project's stretch mode.
+			[b]Note:[/b] Since [InputEventMouseMotion] may only be emitted when the mouse moves, checking for a relative position of [code]Vector2(0, 0)[/code] can't reliably detect when the mouse has stopped moving. A separate [SceneTreeTimer] will likely need to be used to detect that the mouse hasn't moved for some period of time.
+			[b]Note:[/b] This coordinate is [i]not[/i] scaled according to the content scale factor or calls to [method InputEvent.xformed_by]. This should be preferred over [member relative] for mouse aiming when using the [constant Input.MOUSE_MODE_CAPTURED] mouse mode, regardless of the project's stretch mode.
```

Comment:
A nitpick. I personally don't mind either way, but I've seen most of the class reference referencing to constants like this:
```suggestion
			[b]Note:[/b] This coordinate is [i]not[/i] scaled according to the content scale factor or calls to [method InputEvent.xformed_by]. This should be preferred over [member relative] for mouse aiming when using [constant Input.MOUSE_MODE_CAPTURED], regardless of the project's stretch mode.
```

Reply:
This is not my wording. I simply split this "Note" to a new line because it had accidentally been concatenated with the previous "Note". The previous "Note" is the one I was trying to edit. As this wording is replicated elsewhere in the file and unrelated to my change, I'm inclined to leave it as is.